src/main/java/ratpack/pac4j/internal/Pac4jAuthenticator.java

```diff
@@ -36,11 +39,6 @@
 import ratpack.session.SessionData;
 import ratpack.util.Types;
 
-import java.util.List;
-import java.util.Optional;
-
-import static ratpack.util.Exceptions.uncheck;
-
```

Comment:
Changed order of imports.

Reply:
Looks to me like the order of List, Optional, and Exceptions.uncheck still changed.